torch/nn/quantized/modules/conv.py

```diff
@@ -79,6 +86,23 @@ def _init(self, in_channels, out_channels, kernel_size, stride,
         self.scale = 1.0
         self.zero_point = 0
 
+    def _adjust_padding(self):
+        # See `nn._ConvNd` for more details for the implementation.
+        # We have to reimplement it here, to support `torch.jit`
+        # TODO(z-a-f): Factor out common code
+        self._reversed_padding_repeated_twice = [0, 0] * len(self.kernel_size)  # 'valid' padding
```

Comment:
this code seems very similar to the code in `torch.nn.modules.conv._ConvNd.__init__`, can we consider reusing instead of copying?

Reply:
I agree -- I tried doing that, but got `jit` issues that it's not serializable. I added a TODO, but as for resolving it here -- not 100% sure how to resolve it. Should we discuss it in more details?

Reply:
Will post the explanation later -- there was an issue with some functions being compiled, but jit could not handle them.